### PR TITLE
Creating project tags

### PR DESCRIPTION
**Pull request by iparask**
This PR creates `status` categories for projects. When a project's status is indicated as `active` its entry goes to active projects, when indicated as `pas` goes to the past projects.

### CODE REVIEW

**Review by mturilli — APPROVED** [review]
I am going to approve this pull request as my comment took too much time to be considered.

**Comment by mturilli** [comment]
@iparask I would not wait for three reviews before merging this. Note that we should probably merge this before all the other iterations on project descriptions.

File: projects.md [hunk 1]
```diff
@@ -2,22 +2,31 @@
...
 ## Projects
 
 Here you can find currently active and past research projects the Lab was involved with:
 
-### Active Projects
+{% for item in proj_array %}
+<div class="pos_header">
+ {% if item == 'active' %}
+<h3>Active Projects</h3>
+ {% elsif item == 'past' %}
```
[the remark this answers is not in the document]
Comment (iparask) [reply projects.md]: Definetely

**Comment by iparask** [comment]
Okay, I will merge.

**Pull request merged**